app/models/pydantic/assets.py

```diff
@@ -25,9 +25,14 @@ class AssetCreateIn(BaseModel):
     asset_type: AssetType
     asset_uri: Optional[str]
     is_managed: bool = True
-    creation_options: CreationOptions
+    creation_options: OtherCreationOptions
+    fields_value: Optional[List[Dict[str, Any]]]
```

Comment:
Is this b/c of the dynamic vector tile cache?
https://github.com/wri/gfw-data-api/blob/develop/app/routes/tasks/task.py#L283

We already have two very similar parameters defined in pydantic/creation_options 
- `field_attributes`
- `table_schema`

`field_attributes` is also used for the Static vector tile cache and would the better choice.
https://github.com/wri/gfw-data-api/blob/develop/app/models/pydantic/creation_options.py#L159-L163

Reply:
It was indeed.
Just checking, @thomas-maschler you're suggesting adding `field_attributes` to the creation_options of the dynamic vector tile cache model and removing this from the top level of the AssetCreateIn model, right?

Reply:
Done.